src/mmw/apps/modeling/views.py

```diff
@@ -567,6 +567,25 @@ def drb_point_sources(request):
                     headers={'Cache-Control': 'max-age: 604800'})
 
 
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def kill_job(request, job_uuid, format=None):
+    try:
+
+        celery.current_app.control.revoke(job_uuid,
+                                          terminate=True,
+                                          signal='SIGUSR1')
```

Comment:
The reason to send the `SIGUSR1` signal instead of `SIGTERM` or `SIGKILL` is that celery will raise a `SoftTimeLimitExceeded` for the former, whereas `SIGTERM` and `SIGKILL` both appear to immediately kill the process. I would have expected celery to raise a `HardTimeLimitExceeded` exception upon a `SIGTERM` signal but it doesn't appear to be the case.

The reason for use of `SIGUSR1` is evident through reading the docs, but I can add a comment here if it would be useful.